doc/node-props.md

```diff
@@ -0,0 +1,11 @@
+# Properties of nodes
+These are properties babel stores in AST node objects for internal use, as opposed to properties that are part of the AST spec (ESTree at the time of this writing).
+
+## `_blockHoist`
+`node._blockHoist != null` triggers the [block-hoist transformer](/src/babel/transformation/transformers/internal/block-hoist.js). Value should be `true` or an integer in the range `0..3`. `true` is equivalent to `2`. The value indicates whether the node should be hoisted and to what degree. See the source code for more detailed information.
```

Comment:
@hzoo The rich diff is the feature that links incorrectly. The "whole file" view also works for me. Likely due to the whole file view being at `https://github.com/jmm/babel/blob/internals-docs2/doc/node-props.md`, not in `github.com/babel/babel`.